src/modules/buildDiscoveryServerSideObj.js

```diff
@@ -5,14 +5,14 @@ import{extractMetaTagContent} from './extractMetaTagContent';
 export let buildObject = (watermarkPresent) => {
     let gtmDL;
     if (watermarkPresent) {
-        gtmDL = Object.assign(watermarkObject('ivp', extractMetaTagContent('DCSext\\.imgviewer', 'Meta tag not available'), 'Image viewer', 'Below record description'), defaultDiscoveryServerSideObject(extractMetaTagContent('WT\\.cg_n', 'Content Group not available'),
-            extractMetaTagContent('DCSext\\.docref', 'Meta tag not available'), extractMetaTagContent('DCSext\\.subscription', 'Subscriber info not available'),
-            extractMetaTagContent('DCSext\\.signedin', 'Registered info not available')));
+        gtmDL = Object.assign(watermarkObject('ivp', extractMetaTagContent('DCSext\\.imgviewer'), 'Image viewer', 'Below record description'), defaultDiscoveryServerSideObject(extractMetaTagContent('WT\\.cg_n'),
```

Comment:
Could we destructure the args for `watermarkObject()`

Reply:
Done